src/platform/baytrail/include/platform/platcfg.h

```diff
@@ -41,4 +41,6 @@
 
 #define LPSRAM_SIZE (PLATFORM_LPSRAM_EBB_COUNT * LPSRAM_BANK_SIZE)
 
+#define LPSRAM_MASK(ignored)	((1 << PLATFORM_LPSRAM_EBB_COUNT) - 1)
```

Comment:
Can you just make it  `LPSRAM_MASK` ? No point to continue adding everywhere argument that is unused. Please also update asm_memory_mangement for Apollolake

Reply:
In newest version I don't modify LPSRAM_MASK anymore, just use it.